### ISSUE TITLE
Website verification only via HTTP even though I have CA installed locally

### ISSUE THREAD

**Issue by lucaswerkmeister** [keybase/keybase-issues#1422]
My website’s certificate is issued by [CAcert](http://www.cacert.org), a Web of Trust-based CA that’s not in most CA collections. When checking my proof (`keybase id lucaswerkmeister`), the client only does the verification over HTTP, even though I have the CAcert certificates installed on my system (e. g., `curl https://lucaswerkmeister.de/keybase.txt` works fine). It would be better if the client made that connection via HTTPS, at least on systems that have the certificate.

**Comment by jomo** [comment]
I'm quite sure the verification isn't done on your system - that would be pointless

**Comment by lucaswerkmeister** [comment]
As far as I understand, the verification is done on the system of *anyone* who wants to verify my identity (i. e., we don’t have to trust Keybase, it just hosts the URL to the gist etc.). And in that case, those who have the CAcert certificates installed could do this verification via HTTPS instead of HTTP. Did I misunderstand something there?

**Comment by jomo** [comment]
True, I thought you were talking about that https verification badge on the website.

**Comment by lucaswerkmeister** [comment]
Yeah, that would still be a HTTP badge.
